misc/python/materialize/cloudtest/app/materialize_application.py

```diff
@@ -101,20 +101,6 @@ def __init__(
 
         self.images = ["environmentd", "clusterd", "testdrive", "postgres"]
 
-        # Label the kind nodes in a way that mimics production.
-        for node in [
-            "cloudtest-control-plane",
-            "cloudtest-worker",
-            "cloudtest-worker2",
-            "cloudtest-worker3",
-        ]:
-            self.kubectl(
-                "label",
-                "--overwrite",
-                f"node/{node}",
-                f"materialize.cloud/availability-zone={node}",
```

Comment:
simplified this by setting AZs [on the nodes themselves](https://github.com/MaterializeInc/materialize/pull/20106/files#diff-c97db4e7aeedf03c5188b371ab66199e525f713dd3741b9fb2718fa6c4af47fa) and decoupling AZ names from the node names

Reply:
That's much nicer, thanks!